### ISSUE TITLE
static val TEST = "test"

### ISSUE THREAD

**Issue by nicholas22** [peichhorn/lombok-pg#26]
The val keyword does not work with static final field declarations, it's not a big deal but would be nice to have ;)

**Comment by peichhorn** [comment]
Yeah for some reason val is only allowed on local declarations (so in methods). Not sure if this can be fixed, but I guess this was intentional. Long story short, it can't hurt to look into it.

I found this old discussion about allowing val on fields: 
http://groups.google.com/group/project-lombok/browse_thread/thread/91749fd8992fe8e2/e38961ba7ed52e4b

So eclipse will be the fun part? Well let's see. ;)
